lib/beaker-pe/install/pe_utils.rb

```diff
@@ -450,7 +450,15 @@ def do_install hosts, opts = {}
               else
                 prepare_host_installer_options(host)
                 register_feature_flags!(opts)
-                generate_installer_conf_file_for(host, hosts, opts)
+
+                if opts[:upgrade] && use_meep?(host['previous_pe_ver'])
```

Comment:
@highb thinking again about the <2016.2.0 cases.  In the real world, we expect recover configuration to still produce a working pe.conf, right?  Ideally we should test that.  The blocking point for us is that we need to put the feature_flag in pe.conf.  What happens if we just create a pe.conf with the feature flag in it and then continue to without a pe.conf specified for puppet-enterprise-installer, and let recover configuration do its thing?